src/app.js

```diff
@@ -170,6 +231,38 @@ app.on('ready', () => {
     }
   }
 
+  function setNotificationTime(event, time) {
+    notificationWindow.close();
+    if (time && time !== store.get('notification.time')) {
+      const hours = time.split(':')[0];
+      const minutes = time.split(':')[1];
+      store.set('notification.time', time);
+      store.set('notification.hours', hours);
+      store.set('notification.minutes', minutes);
+      log.info(`Store updated - time=${time} hours=${hours} minutes=${minutes}`);
+      job.setTime(new CronTime(`0 ${minutes} ${hours} * * *`));
+      job.start();
+    }
+  }
+
+  const job = new CronJob({
+    cronTime: `0 0 20 * * *`,
```

Comment:
The template string here is redundant, can we use single quotes instead?

Reply:
I forgot to replace it, sorry